apps/server/lib/bootstrap.ts

```diff
@@ -261,6 +261,64 @@ export const bootstrap = async (logContext: LogContext, options: any) => {
 		}),
 	);
 
+	await worker.insertCard(
```

Comment:
how do I  make this run only once? I have to keep on reseting the db, to make changes here.

Reply:
@iamsolankiamit You can try using `replaceCard`